### ISSUE TITLE
ply does not show texture

### ISSUE THREAD

**Issue by gpaar** [pro3d-space/PRo3D#416]

**Comment by gpaar** [comment]
Further issue: Upon re-load of a scene that had been stored after importing the said .ply file, the .ply load goes into an infinite loop with thousands of error messages such as 
.....
 8: WARNING: object element "700034.459808" not supported
 8: WARNING: object element "699998.879547" not supported
 8: WARNING: object element "700017.579529" not supported
 8: WARNING: object element "700053.605652" not supported
......

**Comment by haraldsteinlechner** [comment]
There are several things to consider and sub problems:
- Flip Z: this also affects obj files from what i understand. Firstly, i'm not sure whether flip z should be default and what consequences this has. Secondly, the fly home problem is a bug.
- Ply and textures: ply itself specifies texture coordinates but does not contain the texture as such. Maybe there are best practice solution to extending ply with textures (maybe having a diffuse_texture.jpg besides the file).
- Once we used assimp to load any 3d file format. this is suboptimal since it is a native libarary (multiplatform support not easy), old etc. Since for ply we have no explicit loader pro3d falls back to this loader (which messes up some parsing) and fails to provide a solution to textures.
- Long kdtree building time. I suspect this is our old friend, "bad" split parameters. Need to look into this.

Basically it boils down to those approaches:
- implement a different ply loader or fix the problems in interpreting the assimp tree. This also requires to define how textures should be handled.
- not use ply. go via obj wavefront or upcoming gltf support, drop ply from list of available extensions for loading objects.

How important is ply in your workflow?

**Comment by gpaar** [comment]
We will try to generate .obj. It is recommended to add something like "deprecated" to the .ply import menu option
